deploy/manifests/api-service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
+metadata:
+  labels:
+    app.kubernetes.io/component: metrics-adapter
+    app.kubernetes.io/name: prometheus-adapter
+    app.kubernetes.io/version: 0.9.1
```

Comment:
v0.10.0 was released quite recently, would you mind updating it in kube-prometheus and regenerate the manifests so that the manifests that we provide in prometheus-adapter target its latest release? IIRC you would just have to update the value in https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/versions.json#L8 and regenerate the manifests.